source/_integrations/local_ip.markdown

```diff
@@ -21,11 +21,3 @@ local_ip:
 ```
 
 To configure via the user interface, select the `Local IP Address` integration.
-
-{% configuration %}
-name:
```

Comment:
This configuration option is still in the schema for the above-listed parent PR.

Reply:
It has been depretaced now.

Should we remove it from the documentation or mark it deprecated until specific version then ?

Reply:
As long as it can still be used it should be in the documentation, only if it has been removed from the core repo it will also have to be removed in the documentation.

Reply:
I think it is fine. People should not use it anymore.
Furthermore, using it will cause warnings in the Home Assistant system, as the configuration parameters has been marked deprecated in the schema.

We should prevent people "start" using them.